ServiceWebsite/ServiceWebsite/Controllers/ParticipantController.cs

```diff
@@ -34,8 +42,75 @@ public ParticipantController()
         [ProducesResponseType((int)HttpStatusCode.BadRequest)]
         public async Task<IActionResult> UpdateSuitabilityAnswers(Guid hearingId, [FromBody]List<HearingSuitabilityAnswer> answers)
         {
-          await Task.Yield();
-          return NoContent();
+            if (hearingId == Guid.Empty)
+            {
+                return new BadRequestObjectResult($"Please provide a valid {nameof(hearingId)}");
+            }
+            try
+            {
+                var participantId = await _hearingService.GetParticipantIdAsync(User.Identity.Name, hearingId);
+
+                if (!participantId.HasValue || participantId == Guid.Empty)
```

Comment:
you will never have `participant == Guid.Empty` here so you can remove that part

Reply:
Resolved